modules/ROOT/nav.adoc

```diff
@@ -5,6 +5,7 @@
 * xref:dataweave-language-guide.adoc[Language Guide]
  ** xref:dataweave-language-introduction.adoc[Scripts]
  ** xref:dataweave-selectors.adoc[Selectors]
+ ** xref:dataweave-type-selector.adoc[Type Selector]
```

Comment:
Hey, all the other selector doc is in these files: https://docs.mulesoft.com/dataweave/2.4/dataweave-selectors
https://docs.mulesoft.com/dataweave/2.4/dataweave-cookbook-extract-data

Those files are probably too long and could be broken down as child docs, but I think it could be odd to have a special file for the type selector, which works like the single-value selector for other data. So I suspect the doc needs to be incorporated into existing doc or _maybe_ be made a child or something and the other doc get broken down and reorg'd as child docs. Lemme know what you think.